util/logtools/fflogs.ts

```diff
@@ -163,7 +163,7 @@ class FFLogs {
     key: string,
     startTime: number,
     endTime: number,
-    filter = 'source.disposition="enemy" and type="cast"',
+    filter = 'source.disposition="enemy" and (type="cast" or type="begincast")',
```

Comment:
This is a good point, I hadn't considered the need for `test_timeline` when I originally wrote this. But it's perfectly fine already since `make_timeline` skips non-cast lines when using FFLogs data.